### PR TITLE
set traveloggers avatar in user settings

### PR DESCRIPTION
**Pull request by tx0c**

### CODE REVIEW

File: src/components/UserProfile/DropdownActions/EditProfileDialog/Content.tsx [hunk 1]
```diff
@@ -160,6 +200,42 @@ const EditProfileDialogContent: React.FC<FormProps> = ({
...
+                index: number
+              ) => (
+                <TravlogNFTImage
+                  key={nftProps.id}
+                  index={index}
+                  selectedIndex={selectedNFTIndex}
+                  nft={nftProps}
+                  onClick={() => {
+                    setSelectedNFTIndex(index)
+                    setFieldValue('avatar', nftProps.imagePreviewUrl)
```
Comment (tx0c) [inline src/components/UserProfile/DropdownActions/EditProfileDialog/Content.tsx:227]: this imagePreviewUrl looks like `https://server-develop.matters.news/img-cache/https://lh3.googleuser.../original-path` is not a uuid, and server side still need another change for this special case

Comment (tx0c) [reply src/components/UserProfile/DropdownActions/EditProfileDialog/Content.tsx:227]: and thematters/matters-server#2324 is needed to handle this

**Review by guoliu — APPROVED** [review]
🌿

File: src/components/UserProfile/DropdownActions/EditProfileDialog/Content.tsx [hunk 2]
```diff
@@ -160,6 +200,42 @@ const EditProfileDialogContent: React.FC<FormProps> = ({
...
+                  }}
+                />
+              )
+              /* <CachedOpenSeaImg {...{id, imagePreviewUrl}} /> */
+            )}
+            {/* <pre>{JSON.stringify(user.info.cryptoWallet?.nfts)}</pre> */}
+          </section>
+          <footer>點擊圖片替換頭像</footer>
```
Comment (robertu7) [inline src/components/UserProfile/DropdownActions/EditProfileDialog/Content.tsx:235]: @tx0c transalation?

**Pull request merged**